Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7Parser201305Utils.java

```diff
@@ -0,0 +1,123 @@
+/**
+ *
+ */
+package gov.hhs.fha.nhinc.mpi.adapter.component.hl7parsers;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang.StringUtils;
+import org.hl7.v3.CE;
+import org.hl7.v3.PRPAIN201305UV02;
+import org.hl7.v3.PRPAIN201305UV02QUQIMT021001UV01ControlActProcess;
+import org.hl7.v3.PRPAMT201306UV02LivingSubjectAdministrativeGender;
+import org.hl7.v3.PRPAMT201306UV02ParameterList;
+import org.hl7.v3.PRPAMT201306UV02PatientTelecom;
+import org.hl7.v3.PRPAMT201306UV02QueryByParameter;
+import org.hl7.v3.TELExplicit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author PVenkatakrishnan
+ *
+ */
+public class HL7Parser201305Utils {
+
+    private static final Logger LOG = LoggerFactory.getLogger(HL7Parser201305Utils.class);
+    private static HL7Parser201305Utils instance = new HL7Parser201305Utils();
+
+    HL7Parser201305Utils() {
+    }
+
+    public static HL7Parser201305Utils getInstance() {
+        return instance;
+    }
+
+    /**
+     * Method to extract Gender Code from a PRPAMT201306UV02ParameterList.
+     *
+     * @param params the Paramater list from which to extract a Gender Code
+     * @return The Gender Code is returned
+     */
+    public String extractGender(PRPAMT201306UV02ParameterList params) {
+        LOG.trace("Entering HL7Parser201305Utils.ExtractGender method...");
+
+        String genderCode = null;
+
+        // Extract the gender from the query parameters - Assume only one was specified
+        if (CollectionUtils.isNotEmpty(params.getLivingSubjectAdministrativeGender())
+            && params.getLivingSubjectAdministrativeGender().get(0) != null) {
```

Comment:
Sound like checking params.getLivingSubjectAdministrativeGender().get(0) != null is redundancy when calling  CollectionUtils.isNotEmpty(params.getLivingSubjectAdministrativeGender())